.travis.yml

```diff
@@ -0,0 +1,56 @@
+language: go
+
+services:
+  - docker
+
+go:
+  - "1.9.x"
```

Comment:
`1.10.x`?

Reply:
... and `tip` with `allow_failures`, you're right @smola 
I tried, but something was missing in the matrix... and some stages was not run when I tried it...
Could we open an issue with this requirements, merge as it is, and then proceed with it after merging?
If you agree, I'll open the issue to avoid forgeting it.

Reply:
done by https://github.com/src-d/gitbase-playground/pull/15/commits/10fd2d71f694ba5adf7b74804918ce676b32b153